spec/abstract.md

```diff
@@ -16,15 +16,15 @@ publication mechanism. This offers an interoperability bridge from (or between)
 blockchain ecosystems. Also, without directly supporting environments where the
 web is not practical (e.g., IOT, Lo-Ra, BlueTooth, NFC), the method builds on a
 foundation that fully supports those environments, making future interop of
-identifiers between web and non-web an easy step.
+identifiers between web and non-web a manageable step.
```

Comment:
These changes should (I assume) be removed from this PR, since they are already in another. Or I guess that other PR can be cancelled.

Reply:
This will happen automatically, when PR65 is accepted.